modules/installation/proc-installing-with-helm-cli.adoc

```diff
@@ -0,0 +1,57 @@
+[id="installing-with-helm-cli_{context}"]
+= Installing {product} with Helm CLI
+
+As an OpenShift or Kubernetes administrator, you can use the Helm CLI to install {product}.
+
+.Prerequisites
+* An `oc` or `kubectl` session with a user that has `admin` role on an OpenShift or Kubernetes cluster.
+* Helm CLI.
+
+.Procedure
+. Add the OpenShift Helm Charts repository:
++
+----
+$ helm repo add openshift-helm-charts https://charts.openshift.io/
+----
+
+. Create and activate the _<rhdh>_ OpenShift project or Kubernetes namespace:
++
+[subs="quotes+"]
+----
+$ NAMESPACE=_<rhdh>_
+$ oc new-project ${NAMESPACE}
```

Comment:
This will create the new project OR select it, if it already exists. Saves a step. 

```suggestion
$ oc new-project ${NAMESPACE} || oc project ${NAMESPACE}
```